app/javascript/courses/review/components/CoursesReview__Root.re

```diff
@@ -363,16 +379,30 @@ let filterSubmissions = (selectedLevel, selectedCoach, submissions) => {
 };
 
 module Sortable = {
-  type t = sortBy;
+  type t = [ | `EvaluatedAt | `SubmittedAt];
```

Comment:
Sorry, but if the type is defined explicitly somewhere, then do we need polymorphic variant? Couldn't this have been a regular variant? I thought that this was sort of light-weight where we avoid having to define the type explicitly.

Reply:
@mahesh-krishnakumar Correct me if I'm wrong, but, @jasim, I believe the `type t` is required because it's used with the functor `Sorter.Make`...

```reason
module SubmissionsSorter = Sorter.Make(Sortable);
```

...which defines the following `module type`:

```reason
module type Sortable = {
  type t;
  let criterion: t => string;
  let criterionType: t => [ | `String | `Number];
};
```

Since this is _generic_, I can't think of a clean way to avoid having to define `type t` in the call-site. 🤔

Reply:
Got it. Then I guess you don't need a polymorphic variant in the implementation; just a regular variant would do? I don't think it is a big deal anyway - I was asking because I thought there was some polymorphic-variant based technique here I could learn.